nifi-registry-core/nifi-registry-properties/src/main/java/org/apache/nifi/registry/properties/NiFiRegistryProperties.java

```diff
@@ -61,6 +61,7 @@
     public static final String EXTENSION_DIR_PREFIX = "nifi.registry.extension.dir.";
 
     public static final String PROVIDERS_CONFIGURATION_FILE = "nifi.registry.providers.configuration.file";
+    public static final String REGISTRY_ALIAS_CONFIGURATION_FILE = "nifi.registry.registry.alias.configuration.file";
```

Comment:
We also need to update the default properties file that comes in the assembly to have this property

Reply:
Think I've resolved with https://github.com/apache/nifi-registry/pull/162/commits/a2ab149394971b94a1fc571ea340268ed9bc1623